validation-test/compiler_crashers/00695-swift-availabilityattr-isunavailable.swift

```diff
@@ -5,7 +5,7 @@
 // See http://swift.org/LICENSE.txt for license information
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not %target-swift-frontend %s -parse
+// RUN: not --crash %target-swift-frontend %s -parse
```

Comment:
It crashes somewhere in member lookup (I will rename the test) this time around - I'll take a look at it later today.